src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -361,26 +362,33 @@ const TickDisplay = (() => {
         }
     };
 
+    const addSellSpot = (contract) => {
+        // for tick trades only touchnotouch can have sell spot before exit spot
+        if (contract_category !== 'touchnotouch' || !applicable_ticks) return;
```

Comment:
can we find a better way than hardcoding the touchnotouch category? does it break anything if we just remove the contract_category check?

Reply:
@negarn  it may override exit lines for other contracts, but if we only add sell spot if no line was there before, then it should work.